src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
try to use arrow function for avoiding issues with context 
```suggestion
document.addEventListener('DOMContentLoaded', () => {
```